addons/google_calendar/utils/google_event.py

```diff
@@ -65,8 +65,7 @@ def ids(self):
     @property
     def rrule(self):
         if self.recurrence and any('RRULE' in item for item in self.recurrence):
-            rrule = next(item for item in self.recurrence if 'RRULE' in item)
-            return rrule[6:]  # skip "RRULE:" in the rrule string
+            return next(item for item in self.recurrence if 'RRULE' in item)
```

Comment:
I changed this part because it was part of why the code I pushed before had to be so complex.

The problem was that, since it was removing 6 leading characters, sometimes the value passed to `_rrule_parse` was `X-EVOLUTION-ENDDATE=20200120:FREQ=WEEKLY;COUNT=3;BYDAY=MO`, which is totally invalid.

RFC also states that there are valid IANA parameters that could exist before the colon.

Thus I decided to just save the whole RRULE, including the prefix. I had to change some tests accordingly. Dateutil parses it correctly with or without the `RRULE:` prefix when it's well formed, so it doesn't seem to be a problem.